projects/vdk-core/src/vdk/internal/core/errors.py

```diff
@@ -607,32 +363,167 @@ def __error_type_to_actual_resolver(to_be_fixed_by: ResolvableBy) -> ResolvableB
     )  # What type is the error that caused this and whom to blame, Platform or Data Jobs Developer?
```

Comment:
I think we can remove this and just default to platform error. Even if we reach this case, we'll need more info on what exactly happened, so it makes sense for it to be a platform error.